app/assets/stylesheets/earthworks.css

```diff
@@ -317,3 +317,29 @@ label.toggle-bookmark {
   --bs-link-color-rgb: var(--stanford-digital-blue-dark-rgb);
   text-decoration: underline;
 }
+
+/* Override mobile view for facets section in Blacklight */
+.facets {
+  --bl-facets-smallish-border: none;
+  --bl-facets-smallish-padding: 0;
+}
+
+/* To help vertically align on mobile */
+@media (max-width: 991.98px) {
+  .facets-heading {
+    margin-bottom: 0;
+  }
+}
+
+/* Hide/show button on mobile */
+.facets-header .navbar-toggler {
+  --bl-facets-smallish-border: none;
+  --bs-navbar-toggler-border-color: var(--stanford-digital-blue);
+  --bs-navbar-color: var(--stanford-digital-blue);
+  color: var(--stanford-digital-blue);
```

Comment:
is this `color` necessary? This rule should apply:
```
.navbar-toggler {
...
	color: var(--bs-navbar-color);
}
```

Reply:
For whatever reason, it doesn't work with just the "--bs-navbar-color" definition.The "color" property is set explicitly elsewhere, so --bs-navbar-color isn't the only thing determining the color.